__init__.py

```diff
@@ -476,6 +482,41 @@ def on_pairing_end(self, error_dialog):
         self.state = SetupState.INACTIVE
 
     # Pairing GUI events
+    ### Language Selection & Welcome Screen
+    def handle_welcome_screen(self):
+        self.state = SetupState.WELCOME
+        self.handle_display_manager("Welcome")
+        sleep(0.3) # Wait for display to show the screen before speaking
+        self.speak_dialog("welcome_screen", wait=True)
+        sleep(0.3) # Wait for display a bit before showing the next screen
+        self.handle_language_menu()
```

Comment:
need to add a check for PairingMode enum and skip this if using voice mode, with voice mode we can not select language